rust/CMakeLists.txt

```diff
@@ -18,5 +12,11 @@ add_test(
 
 add_test(
     NAME rs-tests
-    COMMAND cargo test --manifest-path ${CMAKE_CURRENT_SOURCE_DIR}/Cargo.toml --target-dir ${CMAKE_CURRENT_BINARY_DIR} -p test_fracpack
+    COMMAND cargo test --manifest-path ${CMAKE_CURRENT_SOURCE_DIR}/Cargo.toml --target-dir ${CMAKE_CURRENT_BINARY_DIR} -p test_fracpack -p psibase -p cargo-psibase
```

Comment:
WDYT about adding `test_contract` and `test_contract_2` here too (although it needs to be executed from `cargo psibase test`)? I think it would be nice since these two services are our main examples that we always keep up-to-date (or at least try to).